nectar/config.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
+from builtins import object
 
 import os
 import tempfile
 
 import requests.auth
 
+temp_files = []
```

Comment:
Hello.

Maybe it is useless change but without it, I got this error message in tests result in both Pythons:

```
...........Exception ignored in: <bound method DownloaderConfig.__del__ of <nectar.config.DownloaderConfig object at 0x7f9588184e80>>
Traceback (most recent call last):
  File "/home/lbalhar/Software/nectar/nectar/config.py", line 187, in __del__
    self.finalize()
  File "/home/lbalhar/Software/nectar/nectar/config.py", line 197, in finalize
    for file_name in self._temp_files:
AttributeError: 'DownloaderConfig' object has no attribute '_temp_files'
```

This exception is ignored but with my change, it doesn't appear.

Should I revert nectar/config.py to the previous state?